README.md

```diff
@@ -37,46 +39,49 @@ $ npm run build
 
 ## Principles
 
-* **One dependency:** The first and only dependency needed to create a Cycle.js project, hiding tooling complexity and providing smart defaults. Just update `create-cycle-app` to get selected changes to the core flavors.
+* **One dependency:** The only dependency needed to create a Cycle.js project, hiding tooling complexity and providing smart defaults. Just update `create-cycle-app` to get selected changes to the core flavors.
 
-* **Zero Configuration:** There are no configuration files. Configuring both development and production builds is handled for you so you can focus on writing code.
+* **Zero Configuration:** No configuration files. Configuring both development and production builds is handled for you. Focus on writing code.
 
 * **Many Flavors:** We like to be together not the same, that’s why create-cycle-app comes with 1 core flavors but allows you to provide your own from any registry such as GitHub or your own.
 
-* **No Lock-In:** Specifically made for beginners and to provide fast bootstrap for new projects, create-cycle-app doesn't have the ambition to be __the__ tool for working with Cycle.js projects. With that in mind, it's easy to leave `create-cycle-app` defaults and follow your own steps, by running `npm run eject`.
+* **No Lock-In:** Specifically made for beginners and to provide fast bootstrap for new projects. `create-cycle-app` doesn't have the ambition to be __the__ tool for working with Cycle.js projects. With that in mind, it's easy to leave `create-cycle-app` defaults and follow your own steps, by running `npm run eject`.
 
 ## Why Use This?
 
 **If you’re getting started** with Cycle.js, use `create-cycle-app` to automate the build of your app. There is no configuration file, and `cycle-scripts-<flavorName>` is the only extra build dependency in your `package.json`. Your environment will have everything you need to build a Cycle.js app.
 
 **If you’re a power user** simply use it as a boilerplate generator, by passing in your own flavor.
 
+**To customise** the build or compile process, you need to first `eject` the dependencies and configurations into the project root folder. Then you can customize from there as usual.
 
 ## Getting Started
 
 ### Installation
 
-Installing globally provides a create-cycle-app command for creating new projects.
+Install the module globally and a `create-cycle-app` binary command will be made available for creating new projects.
```

Comment:
I guess we could remove some jargon and make it clearer that you will be able to use that command on your terminal perhaps?